### PR TITLE
Signin page

### PR DESCRIPTION
**Pull request by shemaeric**
### What does this Pull Request do?

This Pull request applies all the functionalities of the `HTML FORM project`.

### CODE REVIEW

**Review by stratospherique — COMMENTED** [review]
### Pull request under review by Ahmed Mahfoudh TSE

**Review by stratospherique — CHANGES_REQUESTED** [review]
### Great work team :clap: . However, you need to implement some requirements in order for us to approve your project. check below :arrow_down: 

#### Page structure

##### Sign In Button

* [ ] It has a Sign In button at the top right corner.

##### Intuit products

* [ ] The products row is under the Intuit logo and above the form.
* [x] It shows the Mint logo and text (logo on the left, text on the right).
* [ ] It shows the Quickbooks logo and text (logo on the left, text on the right).
* [ ] It shows the Turbotax logo and text (logo on the left, text on the right).

##### Form

###### Create account button

- **[REQUIRED]** add an attribute `type="submit"` to your button :nerd_face: 

### General requirements

- **[REQUIRED]** Your linter checker is not properly configured. Did you check [Microverse's guide](https://github.com/microverseinc/linters-config/tree/master/css) ? Please do and follow the instruction :small_airplane: 

### That's it :relieved: . When finished, kindly resubmit another code review request :pray: .

**Comment by shemaeric** [comment]
> ### Great work team 👏 . However, you need to implement some requirements in order for us to approve your project. check below ⬇️
> #### Page structure
> ##### Sign In Button
> * [ ]  It has a Sign In button at the top right corner.
> 
> ##### Intuit products
> * [ ]  The products row is under the Intuit logo and above the form.
> * [x]  It shows the Mint logo and text (logo on the left, text on the right).
> * [ ]  It shows the Quickbooks logo and text (logo on the left, text on the right).
> * [ ]  It shows the Turbotax logo and text (logo on the left, text on the right).
> 
> ##### Form
> ###### Create account button
> * **[REQUIRED]** add an attribute `type="submit"` to your button 🤓
> 
> ### General requirements
> * **[REQUIRED]** Your linter checker is not properly configured. Did you check [Microverse's guide](https://github.com/microverseinc/linters-config/tree/master/css) ? Please do and follow the instruction 🛩
> 
> ### That's it 😌 . When finished, kindly resubmit another code review request 🙏 .

Thank you @stratospherique for the review. We have applied all the requested changes, you can look at it again.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+              Standard call, messaging or data rates may apply.
+            </p>
+          </div>
+
+          <div class="input-box">
+            <label for="password"> Password <span></span></label>
+            <input type="password" name="password" id="password" required />
+          </div>
+          <div class="input-button">
+            <button name="create"><span class="custom-span"></span> Create Account</button>
```
Comment (whiz25) [inline index.html]: please change this to ```<input type='submit' value='Create Account>```. this was already pointed out by the previous reviewer. make only this one change and this project will get approved.

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Good job :+1: 
Check my comment and make that change and re-submit.

**Review by stratospherique — APPROVED** [review]
### Great work implementing the suggested changes :clap: . Your project is approved :heavy_check_mark:  and you may merge to master :+1: :+1: :+1:

**Pull request merged**